Add qald-9-plus dataset (dbpdia ver) and eval

xixi019 opened the pull request:
Add qald-9-plus dataset (dbpdia ver) and eval based on paper https://arxiv.org/pdf/2202.00120v2.pdf
It turns out this paper: https://dl.acm.org/doi/pdf/10.1145/3485447.3511940 is only tested  on the qald-9-plus wikidata version.

xixi019 committed:
Add qald-9-plus dataset (dbpdia ver) and eval 

Add qald-9-plus dataset (dbpdia ver) and eval based on paper https://arxiv.org/pdf/2202.00120v2.pdf

RicardoUsbeck commented:
@Perevalov what do you think?

Perevalov commented:
@xixi019 Please, refer to this https://ieeexplore.ieee.org/document/9736271 as the QALD-9-Plus paper

RicardoUsbeck commented:
Should we put both - the DBpedia and the Wikidata results - as separate tables there? I mean, we are currently ignoring a paper because looking only at DBpedia, or?

Perevalov commented:
> Should we put both - the DBpedia and the Wikidata results - as separate tables there? I mean, we are currently ignoring a paper because looking only at DBpedia, or?

I think we need DBpedia and Wikidata as separate tables. 
Also the Wikidata-based results from https://dl.acm.org/doi/pdf/10.1145/3485447.3511940 has to be integrated (see figure), it was different evaluation from the original QALD-9-Plus paper. We can mark each result as e.g., `QAnswer-fr-de-Yandex`, which means: question asked in French was translated to German with Yandex and passed to QAnswer

![image](https://user-images.githubusercontent.com/16652575/169980224-08ae2ca6-303c-4429-8c9c-770fe8d2afb4.png)

xixi019 commented:
> Should we put both - the DBpedia and the Wikidata results - as separate tables there? I mean, we are currently ignoring a paper because looking only at DBpedia, or?

My idea is to add another table for wikidata result under [wikidata entry in leaderboard](https://github.com/KGQA/leaderboard#wikidata)

RicardoUsbeck commented:
Yes, let's do both of that

xixi019 committed:
Update part of the result from qald-9-plus paper

Add some results from https://dl.acm.org/doi/pdf/10.1145/3485447.3511940
Rest will be uploaded soon.

RicardoUsbeck commented:
Thanks for the update Xi!  Could you please rename the latest additions to something like "Perevalov et al. (a)" so it is clear without hovering the link, that theses are two different papers?

xixi019 commented:
> Thanks for the update Xi! Could you please rename the latest additions to something like "Perevalov et al. (a)" so it is clear without hovering the link, that theses are two different papers?

Sure, thanks for pointing it out. @Perevalov We talked about adding this table together in ESWC. 
@RicardoUsbeck Could you merge this PR? I have manually added the part of the result from https://github.com/KGQA/leaderboard/pull/35#issuecomment-1135548325, more specifically, row: from "en-native*" to "be-fr-Helsinki NLP"; column: "QAnswer". @Perevalov would add the rest in another PR, he can reuse some tables he had.
